src/translations/fr.json

```diff
@@ -13,7 +13,15 @@
     "returning": "Retour à la base",
     "docked": "A la base",
     "unknown": "Inconnu",
-    "offline": "Déconnecté"
+    "offline": "Déconnecté",
+    "Stopped": "Arrêté",
```

Comment:
These statuses seem like something very custom. I'll implement a better solution for that.